src/main/resources/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilder/help-namespace.properties

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License. You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and limitations under the
+# License.
+text=Enter the namespace to use for the deployment command. If left blank, the namespace \
+     from the manifest will be used if specified, or 'default' if not specified. A namespace may \
```

Comment:
Would rather not duplicate the k8s namespace docs. Linking to the namespace docs is sufficient.